Backend/utils/MentiiLogging.py

```diff
@@ -0,0 +1,30 @@
+import os
+import time
+import logging
+from logging.handlers import TimedRotatingFileHandler
+
+PATH = '../logs/metniilog.log'
```

Comment:
To address this I added a path setting to the config which should be the path to your mentii repo or another location you want mentii files stored. The required config entry is this:
  [MentiiData]
  path: <path>  ex: /git/mentii/mentii